ci/builders/linux_android_debug_engine.json

```diff
@@ -134,6 +134,27 @@
                 ]
             }
         },
+        {
+            "archives": [],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gn": [
+                "--android",
+                "--android-cpu=arm64",
+                "--no-lto",
+                "--enable-vulkan-validation-layers"
+            ],
+            "name": "android_debug_arm64_validation_layers",
```

Comment:
You'll also need a `"cas_archive": false` here.

Reply:
This appears to be what I was missing.  I was thinking having an empty archive field would be the same thing as this.